controllers/v2/user-extension.js

```diff
@@ -0,0 +1,130 @@
+        "externalId": "a1",
+        "roles": [
+            {
+                "_id": "5d7a2d266371783ceb11064e",
+                "code": "SPD",
+                "title": "State Project Director",
+                "immediateSubEntityType": "school",
+                "entities": [
+                    {
+                        "_id": "5da70ff64c67d63cca1b94e0",
+                        "externalId": "3200100201",
+                        "name": "PUNJAB GPS ASPAL KHURD",
+                        "childrenCount": 0,
+                        "entityType": "school",
+                        "entityTypeId": "5d15a959e9185967a6d5e8a6",
+                        "subEntityGroups": [],
+                        "relatedEntities": [
+                            {
+                                "_id": "5da829874c67d63cca1bd9d0",
+                                "entityTypeId": "5d7a290e6371783ceb11064c",
+                                "entityType": "state",
+                                "metaInformation": {
+                                    "externalId": "PBS",
+                                    "name": "Punjab"
+                                }
+                            },
+                            {
+                                "_id": "5da829a94c67d63cca1bd9d2",
+                                "entityTypeId": "5d15a959e9185967a6d5e8ac",
+                                "entityType": "district",
+                                "metaInformation": {
+                                    "externalId": "BARNALA -3",
+                                    "name": "BARNALA"
+                                }
+                            },
+                            {
+                                "_id": "5da829f64c67d63cca1bd9f1",
+                                "entityTypeId": "5d15a959e9185967a6d5e8ab",
+                                "entityType": "block",
+                                "metaInformation": {
+                                    "externalId": "PJ-BARNALA",
+                                    "name": "BARNALA"
+                                }
+                            },
+                            {
+                                "_id": "5da82a1c4c67d63cca1bdae8",
+                                "entityTypeId": "5d15c4ec03cbf959ccabdd2b",
+                                "entityType": "cluster",
+                                "metaInformation": {
+                                    "externalId": "PJ-GPS KALEKE",
+                                    "name": "GPS KALEKE",
+                                    "city": ""
+                                }
+                            }
+                        ]
+                    }
+                ]}]}}
+  */
+
+  /**
+   * Get profile of user.
+   * @method
+   * @name getProfile
+   * @param {Object} req - request data.
+   * @param {String} req.params._id - user id.
+   * @returns {JSON} User profile data. 
+   */
+
+  getProfile(req) {
```

Comment:
@aman-tunerlabs - Why do we need v2, I think we can just add the same info in v1 no ?

Reply:
@aks30  For v1 We are sending related Entities for a single first entities as a related entities plus goaState everything is send, but in v2 we are sending related entities for each entities and ignoring goaState check. Not to break currently using unnati app we are making it seperate.